message.go

```diff
@@ -246,15 +246,16 @@ func (c *Client) GetMessageMetadataPage(ctx context.Context, page, pageSize int,
 
 		Page     int
 		PageSize int
-
-		Sort string
 	}{
 		MessageFilter: filter,
 
 		Page:     page,
 		PageSize: pageSize,
+	}
 
-		Sort: "ID",
+	/* keep default sort to maintain backward compatibility */
+	if req.Sort == "" {
```

Comment:
I would remove this line all together.  I think it better to have a flexible method that does not impose defaults.

I can patch the bridge, which is the main consumer of this code, with these changes.

Reply:
I was not happy with it either, code updated and rebased.